java-frontend/src/main/java/org/sonar/plugins/java/api/JavaFileScannerContext.java

```diff
@@ -58,6 +58,10 @@
 
   List<Tree> getComplexityNodes(Tree tree);
 
+  /**
+   * @deprecated use getComplexityNodes instead
+   */
+  @Deprecated
   List<Tree> getMethodComplexityNodes(ClassTree enclosingClass, MethodTree methodTree);
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)